package.json

```diff
@@ -98,5 +98,11 @@
     "stylelint-config-sass-guidelines": "^3.0.0",
     "stylelint-suitcss": "^1.0.0",
     "terra-toolkit": "^2.0.0"
+  },
```

Comment:
These should be added to the package.json in `packages/terra-base`

Reply:
Ha....oops.  That's what I get for trying to update everything in git.  Sorry about that.